src/components/Popover/PopoverExample.tsx

```diff
@@ -15,7 +14,6 @@ import { NotificationList } from '../NotificationList/NotificationList';
 import { NotificationListItem } from '../NotificationListItem/NotificationListItem';
 import { PopoverBody } from '../PopoverBody/PopoverBody';
 import { PopoverHeader } from '../PopoverHeader/PopoverHeader';
-import { TextPassage } from '../TextPassage/TextPassage';
```

Comment:
I don't know why `tsc` flagged this on my PR rather than on the PR that introduced it, but whatev